control-plane/connect-inject/controllers/pod/pod_controller_test.go

```diff
@@ -602,6 +622,9 @@ func TestProxyConfigurationWrite(t *testing.T) {
 							},
 						},
 					},
+					TransparentProxy: &pbmesh.TransparentProxy{
+						OutboundListenerPort: 15001,
+					},
```

Comment:
Just curious: Newly required when using TProxy?

Reply:
Yes, Iryna changed the mutating webhook for ProxyConfiguration to add this.